pom.xml

```diff
@@ -843,7 +843,7 @@
     <module>distribution</module>
     <module>math-scala</module>
     <module>spark</module>
-    <!--module>flink</module-->
+    <module>flink</module>
```

Comment:
Since is in the works to be fixed with a profile, I'm +1.  That should come relatively soon though- since Flink often breaks the Nightlies and almost always Crashes Hudson.